pkgs/by-name/fa/fahrplan/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, python3, fetchFromGitHub }:
+let
+  pname = "fahrplan";
+  owner = "dbrgn";
+  version = "1.1.2";
+in python3.pkgs.buildPythonApplication {
+  inherit pname version;
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-2QVaA2zqqFuxjkKbDXo+SNzkrDCxXiSYMUOftQ++wO4=";
+  };
+
+  propagatedBuildInputs = with python3.pkgs; [ dateutil requests texttable ];
+
+  meta = with lib; rec {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/208242